controller/api/util/api_utils.go

```diff
@@ -143,6 +144,7 @@ func BuildStatSummaryRequest(p StatsRequestParams) (*pb.StatSummaryRequest, erro
 			},
 		},
 		TimeWindow: window,
+		SkipStats:  p.SkipStats,
```

Comment:
we've since made some slight adjustments to this in master; we've now split this into `StatsSummaryRequestParams` and `StatsBaseRequestParams` - I'd recommend putting SkipStats in `StatsSummaryRequestParams`

Reply:
Sounds good :+1: